build/linux/installer/conf/container.conf

```diff
@@ -1,141 +1,168 @@
-</source>
-
-# MDM metrics from telegraf
-<source>
-  @type tcp
-  tag oms.mdm.container.perf.telegraf.*
-  bind 0.0.0.0
-  port 25228
-  format json
-</source>
-
-# Container inventory
-<source>
-  type containerinventory
-  tag oms.containerinsights.containerinventory
-  run_interval 60
-  log_level debug
-</source>
-
-#cadvisor perf
-<source>
-  type cadvisorperf
-  tag oms.api.cadvisorperf
-  run_interval 60
-  log_level debug
-</source>
-
-<filter kubehealth.DaemonSet.Node**>
-  type filter_cadvisor_health_node
-  log_level debug
-</filter>
-
-<filter kubehealth.DaemonSet.Container**>
-  type filter_cadvisor_health_container
-  log_level debug
-</filter>
-
-#custom_metrics_mdm filter plugin
-<filter mdm.cadvisorperf**>
-  type filter_cadvisor2mdm
-  metrics_to_collect cpuUsageNanoCores,memoryWorkingSetBytes,memoryRssBytes,pvUsedBytes
-  log_level info
-</filter>
-
-<filter oms.mdm.container.perf.telegraf**>
-  type filter_telegraf2mdm
-  log_level debug
-</filter>
-
-<match oms.containerinsights.containerinventory**>
-  type out_oms
-  log_level debug
-  num_threads 5
-  buffer_type file
-  buffer_path %STATE_DIR_WS%/out_oms_containerinventory*.buffer
-  buffer_queue_full_action drop_oldest_chunk
-  buffer_chunk_limit 4m
-  flush_interval 20s
-  retry_limit 10
-  retry_wait 5s
-  max_retry_wait 5m
-</match>
-
-<match oms.api.cadvisorperf**>
-  type out_oms
-  log_level debug
-  num_threads 5
-  buffer_type file
-  buffer_path %STATE_DIR_WS%/out_oms_cadvisorperf*.buffer
-  buffer_queue_full_action drop_oldest_chunk
-  buffer_chunk_limit 4m
-  flush_interval 20s
-  retry_limit 10
-  retry_wait 5s
-  max_retry_wait 5m
-</match>
-
-
-<match kubehealth.DaemonSet**>
-  @type health_forward
-  send_timeout 60s
-  recover_wait 10s
-  hard_timeout 60s
-  heartbeat_type tcp
-  skip_network_error_at_init true
-  expire_dns_cache 600s
-  buffer_queue_full_action drop_oldest_chunk
-  buffer_type file 
-  buffer_path %STATE_DIR_WS%/out_health_forward*.buffer
-  buffer_chunk_limit 3m
-  flush_interval 20s
-  retry_limit 10
-  retry_wait 5s
-  max_retry_wait 5m
-
-  <server>
-    host "#{ENV['HEALTHMODEL_REPLICASET_SERVICE_SERVICE_HOST']}"
-    port "#{ENV['HEALTHMODEL_REPLICASET_SERVICE_SERVICE_PORT']}"
-  </server>
-
-  <secondary>
+  # Fluentd config file for OMS Docker - container components (non kubeAPI)
+
+  # Forward port 25225 for container logs
+  # gangams - not used and get ridoff after confirming safe to remove
+  <source>
+    @type forward
+    port 25225
+    bind 127.0.0.1
+  </source>
+  
+  # MDM metrics from telegraf
+  <source>
+    @type tcp
+    tag oms.mdm.container.perf.telegraf.*
+    bind 0.0.0.0
+    port 25228
+    format json
+  </source>
+
+  # Container inventory
+  <source>
+    @type containerinventory
+    tag oneagent.containerInsights.CONTAINER_INVENTORY_BLOB
+    run_interval 60
+    @log_level debug
+  </source>
+
+  #cadvisor perf
+  <source>
+    @type cadvisor_perf
+    tag oneagent.containerInsights.LINUX_PERF_BLOB
```

Comment:
All these tag changes are unneeded & unrelated to aad/omsagent removal.

Reply:
> All these tag changes are unneeded & unrelated to aad/omsagent removal.
> tag change align to extension config from AMA. in case of aad auth mode, tag will be dcr-<guid>:containerinsights:<BLOB_NAME> . the reason for the new tag names to aligin with aad changes to use same configuration.

Reply:
DCR doesn;t enforce any name, isn;t it ? you have a mapping between fluent tag and datatype/table in mdsd.xml.

Reply:
> DCR doesn;t enforce any name, isn;t it ? you have a mapping between fluent tag and datatype/table in mdsd.xml.

> in case of the DCR, tag  discovered from extension config which will be in this format dcr-:containerinsights:<BLOB_NAME> for corresponding data type. guid part is dynamic and associated to configured workspace..

Reply:
for our 1p DCR, we decide what we want? are you saying you want to match that ? also where is that , can you add that for reference in a file ?

Reply:
> for our 1p DCR, we decide what we want? are you saying you want to match that ? also where is that , can you add that for reference in a file ?
Here is the example - https://jsoneditoronline.org/#left=local.jufewo&right=local.yepomi. The changes will be next PR in stage 3 roll out.